### PR TITLE
feature: Updated documentation for background images

### PR DESCRIPTION
**Pull request by ShrimpCryptid**
- Added new section in `DATA_FORMAT.md` for backdrop images.
  - Link to the markdown document for easier reading: https://github.com/allen-cell-animated/colorizer-data/blob/feature/bg-images/documentation/DATA_FORMAT.md#dataset
- Updated the example dataset to include backdrop images.

Areas for feedback:
- Is there a better way to number/separate subsections? Every time I add a new section, I've had to re-number the other sections, which leads to broken links.

### CODE REVIEW

File: documentation/DATA_FORMAT.md
Comment (ShrimpCryptid) [inline documentation/DATA_FORMAT.md:1]: You can preview the new file here: https://github.com/allen-cell-animated/colorizer-data/blob/feature/bg-images/documentation/DATA_FORMAT.md#dataset

File: documentation/DATA_FORMAT.md [hunk 1]
```diff
@@ -49,17 +49,17 @@ The most important file is the **manifest**, which is a JSON file that describes
...
     "centroids": <relative path to centroids JSON>, //< optional
     "bounds": <relative path to bounds JSON>        //< optional
+    "backdrops": <array of backdrop image sets>     //< optional, see 2. Backdrops for more details
 }
-
-
-*Note: all paths are relative to the location of the manifest file.
 ```
 
+*Note: all paths are relative to the location of the manifest file.*
```
Comment (toloudis) [inline documentation/DATA_FORMAT.md:56]: is this strictly true or are they allowed to be absolute urls also?

File: documentation/DATA_FORMAT.md [hunk 2]
```diff
@@ -196,7 +199,108 @@ The manifest file would look something like this:
...
+
+`manifest.json:`
+
+```txt
+{
+    ...
+    "backdrops": [
+        {
+            "name": <backdrop name>,
+            "key": <backdrop key>, // Must be unique across backdrops. Lowercase alphanumeric characters and underscores only.
```
Comment (toloudis) [inline documentation/DATA_FORMAT.md:218]: maybe worth a general note somewhere near the top that several fields have a "key" attribute which MUST be unique and alphanumeric+underscore.

**Review by toloudis — APPROVED** [review]
LGTM.  I don't know but I assume there must be a markdown symbol for "auto numbering". 🤷

File: documentation/DATA_FORMAT.md (hunk 1 above)
Comment (ShrimpCryptid) [reply documentation/DATA_FORMAT.md:56]: Currently this is strictly true, there's no support for absolute URLs.

**Pull request merged**